include/SFML/Graphics/RectangleShape.hpp

```diff
@@ -93,6 +93,18 @@ class SFML_GRAPHICS_API RectangleShape : public Shape
     ////////////////////////////////////////////////////////////
     Vector2f getPoint(std::size_t index) const override;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the geometric center of the rectangle
+    ///
+    /// The returned point is in local coordinates, that is,
+    /// the shape's transforms (position, rotation, scale) are
+    /// not taken into account.
+    ///
+    /// \return The geometric center of the shape
+    ///
+    ////////////////////////////////////////////////////////////
```

Comment:
If we don't change the doxygen comment, then we might as well just copy the docs from the parent class with:

```
    ////////////////////////////////////////////////////////////
    /// \copydoc sf::Shape::getGeometricCenter()
    ///
    ////////////////////////////////////////////////////////////
```

Reply:
I've tried to follow the existing praxis so if I do this, the comment for the added function will look a bit "rouge".  Would it be ok to go through the doc comments and do this work in a separate commit?

Reply:
If you think the current way is more consistent then let's stick with that